.travis.yml

```diff
@@ -2,15 +2,16 @@ language: scala
 sudo: required
 jdk:
   - oraclejdk8
+addons:
+  apt:
+    packages:
+      - oracle-java8-set-default
```

Comment:
@TimMoore Switches to `jdk: - oraclejdk8` [here](https://travis-ci.org/lakhina/online-auction-java/builds/212837538#L139)
Still failing at [here](https://travis-ci.org/lakhina/online-auction-java/builds/212837538#L192)